CMakeLists.txt

```diff
@@ -138,12 +138,6 @@ set(QUIC_LIBRARY_NAME "msquic" CACHE STRING "Override the output library name")
 
 set(BUILD_SHARED_LIBS ${QUIC_BUILD_SHARED})
 
-# FindLTTngUST does not exist before CMake 3.6, so disable logging for older cmake versions
```

Comment:
Removing this, as we require 3.16 at minimum higher up, so this can never be hit.